src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -3805,8 +3906,12 @@ def try_create_application_insights(cmd, functionapp):
                    'You can visit https://portal.azure.com/#resource%s/overview to view your '
                    'Application Insights component', appinsights.name, appinsights.id)
 
-    update_app_settings(cmd, functionapp.resource_group, functionapp.name,
+    if not is_centauri_functionapp(cmd, ai_resource_group_name, ai_name):
```

Comment:
Any reason why we use connection string for centauri app here?

Reply:
That's the app setting the backend team asked us to set on our end: [sample](https://microsoft.sharepoint.com/teams/Antares/Shared%20Documents/Forms/AllItems.aspx?csf=1&web=1&e=Eyf2Cj&cid=2a215fa2%2De997%2D4e52%2D839d%2Ded610227dd5d&FolderCTID=0x0120003D52F2E062EFA04BBA4FE3B7752B69D5&id=%2Fteams%2FAntares%2FShared%20Documents%2FFunctions%2FCentauri%2FDesign%20docs%2FAPI%20contracts%2FSiteContract%2Ejson&parent=%2Fteams%2FAntares%2FShared%20Documents%2FFunctions%2FCentauri%2FDesign%20docs%2FAPI%20contracts)

It looks like they do not want `APPINSIGHTS_INSTRUMENTATIONKEY` but rather `APPLICATIONINSIGHTS_CONNECTION_STRING`